e2e/cypress/integration/enterprise/cloud/billing/subscriptions_free_trial_spec.js

```diff
@@ -0,0 +1,260 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @cloud_only @cloud_trial
+// Skip:  @headless @electron // run on Chrome (headed) only
+
+describe('System Console - Subscriptions section', () => {
+    before(() => {
+        // * Check if server has license for Cloud
+        cy.apiRequireLicenseForFeature('Cloud');
+
+        cy.apiInitSetup().then(({team}) => {
+            cy.visit(`/${team.name}/channels/town-square`);
+        });
+
+        gotoSubscriptionScreen();
```

Comment:
`cy.apiInitSetup` is not necessary since it's not testing anything related to team/user. Also, just go straight using admin URL, since we've already test in place that navigates to admin section. Something like:
```suggestion
        // # Visit Subscription page
        cy.visit('/admin_console/billing/subscription');
        cy.get('.admin-console__header').should('be.visible').and('have.text', 'Subscription');
```

Reply:
gotoSubscriptionScreen() has been removed and updated script as per feed back comment.